packages/gatsby-plugin-typescript/README.md

```diff
@@ -21,6 +21,8 @@ module.exports = {
 }
 ```
 
+When [creating pages programmatically](https://www.gatsbyjs.org/docs/programmatically-create-pages-from-data/#creating-pages), you can pass the `.tsx` filename directly as the `component` for [`createPage`](https://www.gatsbyjs.org/docs/actions/#createPage).
```

Comment:
Seeing as this is suggested but not explicitly stated, I would rather add this into the doc as a note. 
```suggestion
 > When [creating pages programmatically](/docs/programmatically-create-pages-from-data/#creating-pages), you can pass the `.tsx` filename directly as the `component` for [`createPage`](/docs/actions/#createPage).
```